modules/oe_theme_content_organisation/oe_theme_content_organisation.module

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/**
+ * @file
+ * Module file used for theming the Organisation content type.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Cache\CacheableMetadata;
+use Drupal\Core\Entity\Entity\EntityViewDisplay;
+use Drupal\file\FileInterface;
+use Drupal\image\Plugin\Field\FieldType\ImageItem;
+use Drupal\media\MediaInterface;
+use Drupal\media_avportal\Plugin\media\Source\MediaAvPortalPhotoSource;
+use Drupal\media\Plugin\media\Source\Image;
+use Drupal\oe_theme\ValueObject\ImageValueObject;
+
+/**
+ * Prepares variables for in-page navigation.
+ *
+ * Default template: oe-theme-helper-in-page-navigation.html.twig.
+ *
+ * @param array $variables
+ *   An array of variables.
+ */
+function oe_theme_content_organisation_preprocess_oe_theme_helper_in_page_navigation(array &$variables): void {
+  /** @var \Drupal\node\NodeInterface $node */
+  $node = $variables['entity'];
+
+  if ($node->get('oe_organisation_logo')->isEmpty()) {
+    return;
+  }
+
+  /** @var \Drupal\media\Entity\Media $media */
+  $media = $node->get('oe_organisation_logo')->entity;
+
+  if (!$media instanceof MediaInterface) {
+    // The media entity is not available anymore, bail out.
+    return;
+  }
+
+  $source = $media->getSource();
+
+  // If its an image media, render it and assign it to the image variable.
+  if (!$source instanceof MediaAvPortalPhotoSource && !$source instanceof Image) {
+   return;
+  }
+
+  $cacheability = CacheableMetadata::createFromRenderArray($variables);
+  $cacheability->addCacheableDependency($media);
+  $thumbnail = !$media->get('thumbnail')->isEmpty() ? $media->get('thumbnail')->first() : NULL;
+
+  if (!$thumbnail instanceof ImageItem || !$thumbnail->entity instanceof FileInterface) {
+    $cacheability->applyTo($variables);
+    return;
+  }
+
+  $variables['image'] = ImageValueObject::fromStyledImageItem($thumbnail, 'oe_theme_ratio_3_2_medium');
+
+  $cacheability->addCacheableDependency($thumbnail->entity);
+  $cacheability->addCacheableDependency(\Drupal::entityTypeManager()->getStorage('media_type')->load($media->bundle()));
+  $cacheability->addCacheableDependency(EntityViewDisplay::collectRenderDisplay($media, 'oe_theme_ratio_3_2_medium'));
+  $cacheability->applyTo($variables);
```

Comment:
I think it would be useful to create an image formatter that would format an image using a media container, given its image style settings.

@drishu I'll create a follow up, so then we can refactor this using that formatter, at which point here we can just do:

```php
    $variables['image'] = $view_builder->viewField($entity->get('field_test'), [
      'type' => 'oe_theme_helper_media_container_formatter',
      'label' => 'hidden',
      'settings' => [
        'image_style' => 'oe_theme_ratio_3_2_medium',
      ],
    ]);
```
And in the template:

```twig
{% block navigation %}
<div class="ecl-col-12 ecl-col-lg-3">
  {% if image %}
    {{ image|field_value }}
  {% endif %}
  {% include '@ecl-twig/inpage-navigation' with {
    'title': title,
    'links': _items,
    'icon_path': ecl_icon_path
  } %}
</div>
{% endblock %}

```
```